swift/ql/lib/codeql/swift/generated/stmt/BraceStmt.qll

```diff
@@ -40,5 +41,38 @@ module Generated {
      * Gets the number of elements of this brace statement.
      */
     final int getNumberOfElements() { result = count(int i | exists(this.getElement(i))) }
+
+    /**
+     * Gets the `index`th variable declared in the scope of this brace statement (0-based).
+     *
+     * This includes nodes from the "hidden" AST. It can be overridden in subclasses to change the
+     * behavior of both the `Immediate` and non-`Immediate` versions.
+     */
+    VarDecl getImmediateVariable(int index) {
```

Comment:
Is this going to conflict with https://github.com/github/codeql/pull/13249 ?

Reply:
yes definitely, you should rebase this on top of `codeql-cli-2.13.3`